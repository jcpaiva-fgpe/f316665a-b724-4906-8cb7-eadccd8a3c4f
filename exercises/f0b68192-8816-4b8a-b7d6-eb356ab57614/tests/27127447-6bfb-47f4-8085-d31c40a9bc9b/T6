17 22
1110001110011100011100
0101000101001010001010
1110011110011100111100
0011110011100111100111
1010001110010100011100
1110001010101100011100
0001000101000010001010
1110010110011100101100
0010110011100101100111
1010001110010100011100
1010001011100101011100
1110001010111100010101
1110001010011100010100
0010001011000100010110
0110001010001100010100
0110001010001100010100
0110001010000101110100
