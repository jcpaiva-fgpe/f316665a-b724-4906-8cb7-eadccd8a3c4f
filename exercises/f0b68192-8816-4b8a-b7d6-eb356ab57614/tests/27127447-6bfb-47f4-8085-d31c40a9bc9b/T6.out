1110001010001100011100
